tools/deploy-to-jetpack-built-git-repo.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+JETPACK_GIT_DIR=$(dirname "$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )" )
+JETPACK_TMP_DIR="/tmp/jetpack"
+JETPACK_TMP_DIR_2="/tmp/jetpack2"
+
+cd $JETPACK_GIT_DIR
+
+# Make sure we don't have uncommitted changes.
+if [[ -n $( git status -s --porcelain ) ]]; then
+ 	echo "Uncommitted changes found."
+ 	echo "Please deal with them and try again clean."
+ 	exit 1
+fi
+
+read -p "You are about to deploy a new built to the jetpack-built repo. Are you sure? [y/N]" -n 1 -r
```

Comment:
Should we update this language to say "jetpack-built" branch now that @dereksmart updated this PR to push to the `jetpack-built` branch in this repo?